js/util/vectortile_to_geojson.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var VectorTileFeature = require('vector-tile').VectorTileFeature;
+
+module.exports = Feature;
```

Comment:
I feel weird about creating a GeoJSON object with a constructor. How would you feel about using `Object.create` to get the prototype perf goodness with an interface like `convertVectorFeatureToGeoJSON(vectorTileFeature, z, x, y)`?